config.json

```diff
@@ -28,7 +28,7 @@
       }
     },
     "pnpm": {
-      "default": "6.29.1",
+      "default": "7.1.0",
```

Comment:
Ping @zkochan @aduh95, I'm not sure whether we can/should/must upgrade the default pnpm from 6 to 7, what do you think (I'm worried it would make it a breaking change for Node)?

Also cc @mcollina: with your proposal (automatically retrieving the latest versions), in a situation like this one, do you think Corepack should just upgrade from 6 to 7?

Reply:
I think we should aim for the latest, Corepack is still experimental in Node.js so breaking changes are still expected (and the upside is worth the potential breakage imo).

I think Matteo proposal was we should use a semver-range as the default (so `7.x` in this instance), so on a fresh install of Node.js you're less likely to get a version that contains known security issues.

Reply:
I don't know, whatever the Node.js team thinks is appropriate.
But do update pnpm to v7 in Node.js 18.

Reply:
It should use ^ by default, meaning that updating a major should be considered a breaking change. Exceptions should also be possible, like for npm 8, where the only breaking change wrt 7 was dropping node 10.